timebot/timebot.go

```diff
@@ -3,10 +3,24 @@
 package timebot
 
 import (
+	"fmt"
 	"time"
+	"strings"
 )
 
 // ParseTime takes a string and returns time.Time in time.UTC
 func ParseTime(text string) (time.Time, bool) {
-	return time.Time{}, true
+	const longForm = "2006-01-02 15:04 -0700"
+
+	text = strings.Replace(text, "PST", "-0800", 1)
+	text = strings.Replace(text, "KST", "+0900", 1)
+	
+	isParsed := true
+	t, err := time.Parse(longForm, text)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
이건 마이너해서 굳이 변경 안하셔도 됩니다
그냥 논의를 위해 일단 던져봅니다

golang에서는 간결한걸 최대한 추구하는만큼 
바로 return하는 것이 좀더 golang에서 idiomatic 한것 같습니다. 

`isParsed` 변수 생성도 필요 없어지구요

```suggestion
		return t, false
```

Reply:
작은 부분인데 무지 매력적이란 생각이 듭니다. :-)